src/v/kafka/protocol/schemata/flex_api_version_generator.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2022 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+#
+# Flexible version code generator for kafka API
+# =============================================
+#
+# Much of the work with flexible versions depends on there being a mapping of
+# each API to the first version that it had introduced flex support.
+#
+# This program creates these lookup tables for other C++ and python programs
+# to consume
+
+import io
+import sys
+import os
+import json
+import pathlib
+import jinja2
+
+
+class VersionRange:
```

Comment:
Some code duplication here, this class exists in `generator.py` as well. I struggled with getting `pex` to act like a proper project and be able to import some utility file, maybe devprod can help